rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/ast/statements.kt

```diff
@@ -6,7 +6,8 @@ import com.smeup.rpgparser.parsing.parsetreetoast.acceptBody
 import com.smeup.rpgparser.parsing.parsetreetoast.toAst
```

Comment:
`import com.smeup.rpgparser.interpreter.*` - Is not good.

Signature of Statement.execute is erroneous for these reasons:
1) `execute(internal_interpreter: InternalInterpreter)` - the naming convention is not appropriate, argument name should be: internalInterpreter not internal_interpreter;
2) internalInterpreter correct argument type should be: `InternalInterpreterCore `not `InternalInterpreter`

statement.kt should knows interpreter, only through interface not through concrete implementation

Reply:
ok